### ISSUE TITLE
Layout/ArgumentAlignment - False positive?

### ISSUE THREAD

**Issue by dmarcoux** [sds/haml-lint#333]
The following code is in a `.js.haml` file in my Rails application and somehow, it's triggering an offense for the cop `Layout/ArgumentAlignment`:

https://github.com/openSUSE/open-build-service/blob/94660dfcced5541ece9ed3c964c779896ad051b6/src/api/app/views/webui/interconnects/create.js.haml#L5

I kept the exact same indentation as in my file. This is what RuboCop outputs:

`Layout/ArgumentAlignment: Align the arguments of a method call if they span more than one line.`

After trying out various indentation changes, I just don't get it. Someone has a clue?

## RuboCop and haml-lint version

```
# Rubocop
1.16.1 (using Parser 3.0.1.1, rubocop-ast 1.7.0, running on ruby 2.5.8 x86_64-linux-gnu)
  - rubocop-performance 1.10.2
  - rubocop-rails 2.10.1
  - rubocop-rspec 2.2.0

# haml-lint
0.37.0
```

**Comment by sds** [comment]
Thanks for the report! 

This is fixed in [0.37.1](https://rubygems.org/gems/haml_lint/versions/0.37.1).

**Comment by dmarcoux** [comment]
Thank you for looking into the issue. Somehow, after updating to 0.37.1, I still get the same offenses. It seems like my haml-lint configuration isn't relying on the default configuration. I do have [a list of ignored RuboCop cops in my haml-lint configuration](https://github.com/openSUSE/open-build-service/blob/5c4fc9123681c54617fd3e16b8066d205df15cdb/src/api/.haml-lint.yml#L12-L36), so perhaps it's overwriting what is defined in the default configuration. Anyway, I ended up also ignoring the `Layout/ArgumentAlignment` cop in my configuration.
